dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/SSLTests15.java

```diff
@@ -14,18 +14,30 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import org.junit.AfterClass;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import componenttest.custom.junit.runner.FATRunner;
 
 @RunWith(FATRunner.class)
 public class SSLTests15 extends SSLCommonTests {
-    @AfterClass
-    public static void stopTestServer() throws Exception {
+
+    @After
+    public void stopTestServer() throws Exception {
+        String methodName = testName.getMethodName();
         if (!javaVersion.startsWith("1.")) {
-            server.stopServer("CWWKC0265W");
+            if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+                super.stopServer(true, "CWWKC0265W", "CWWKT0015W");
+            } else {
+                super.stopServer(true, "CWWKC0265W");
+            }
+        } else {
+            if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+                super.stopServer(true, "CWWKT0015W");
+            } else {
+                super.stopServer();
```

Comment:
why not `stopServer(true);`

Reply:
super.stopServer() will call super.stopServer(true)